specifications/10-flexible-rollout-strategy.json

```diff
@@ -155,13 +171,21 @@
             "expectedResult": true
         },
         {
-            "description": "Feature.flexibleRollout.0 should be disbaled for rollout=0",
+            "description": "Feature.flexibleRollout.0 should be disabled for rollout=0",
             "context": {
                 "sessionId": "147",
                 "userId": "12"
             },
             "toggleName": "Feature.flexibleRollout.0",
             "expectedResult": false
+        },
+        {
+            "description": "Feature.flexibleRollout.customStickiness.42 should be enabled for organization=147",
+            "context": {
+                "organization": "147"
+            },
+            "toggleName": "Feature.flexibleRollout.0",
+            "expectedResult": false
```

Comment:
Description says "...should be enabled" while expected result is `false`?

Reply:
oups... fixed